Dockerfile

```diff
@@ -36,12 +41,14 @@ ENV PATH=/opt/conda/bin:$PATH
 # install js dependencies
 RUN npm install -g configurable-http-proxy && rm -rf ~/.npm
 
-ADD . /srv/jupyterhub
-WORKDIR /srv/jupyterhub/
+ADD . /src/jupyterhub
+WORKDIR /src/jupyterhub
 
 RUN python setup.py js && pip install . && \
-    rm -rf node_modules ~/.cache ~/.npm
+    rm -rf node_modules ~/.cache ~/.npm $PWD
```

Comment:
I don't think ``node_modules`` is strictly necessary after adding ``$PWD``. I'm assuming you are wanting to clear out all cruft from the PWD. Any reason why? Container size?